src/scripts/TranscriptsTrack.js

```diff
@@ -1349,7 +2661,49 @@ const TranscriptsTrack = (HGC, ...args) => {
           });
         });
 
-      this.renderTextBg(this.allBoxes, this.allTexts, allTiles);
+      switch (this.options.blockStyle) {
+        case "directional":
+        case "UCSC-like":
+          this.renderTextBg(this.allBoxes, this.allTexts, allTiles);
+          break;
+        case "boxplot":
+          //this.renderDirectionlessTextBg(this.allBoxes, this.allTexts, allTiles);
+          break;
+        default:
+          throw new Error(
+            'Uncaught TypeError: Unknown blockStyle option (drawLabels, within tile data, B)'
+          );
+      }
+    }
+
+    renderDirectionlessTextBg(allBoxes, allTexts, allTiles) {
```

Comment:
Looks like this isn't used?

Reply:
I added a `colorBoxplotLabel` track option in commit 85cb079, which is a Boolean flag to render labels with a color background that matches the boxplot element coloring. 

The default setting for this option is `false`. If the flag is `true`, the `renderDirectionlessTextBg` function is used to render this background. The label text color will change based on the boxplot element coloring.